micropy/config/config.py

```diff
@@ -105,6 +105,28 @@ def merge(self, config: Union[dict, 'Config'], sync: bool = True) -> dict:
             return self.sync()
         return self.config
 
+    def parse_key(self, key: str) -> Tuple[Sequence[str], str]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/BradenM/micropy-cli/pullRequest?prid=4593453)